powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/ApplicationController.java

```diff
@@ -62,9 +65,11 @@ public ApplicationController( ApplicationServiceBehavior applicationService) {
      */
     @PostMapping("/list")
     public ObjectResponse<GetApplicationListResponse> getApplicationList() throws Exception {
-        logger.info("GetApplicationListRequest received");
+        logger.info("action: getApplicationList, state: initiated");
+        logger.debug("action: getApplicationList, state: initiated, empty request");
```

Comment:
I suggest this change to make the parsing convenient even though there is no rich information to share.
```suggestion
        logger.debug("action: getApplicationList, state: initiated, request: empty");
```